### DISCUSSION TITLE
Demo Collection

### DISCUSSION THREAD

**Discussion by zhensherlock** [zhensherlock/watermark-js-plus#590] [Announcements]
webpack: https://stackblitz.com/edit/webpack-webpack-js-org-dysy1x
react: https://stackblitz.com/edit/vitejs-vite-mdnntv?file=src%2FApp.tsx
heroui: https://stackblitz.com/edit/watermark-js-plus-in-heroui
nextjs: https://stackblitz.com/edit/stackblitz-starters-pv2mvx?file=package.json
remix: https://stackblitz.com/edit/remix-run-remix-q4jgus?file=app%2Froutes%2F_index.tsx
vue2: https://stackblitz.com/edit/vue2-vite-starter-t8cew5
vue3: https://stackblitz.com/edit/vitejs-vite-m54xz7?file=src%2FApp.vue
quasar: https://stackblitz.com/edit/stackblitz-starters-rxyjhx
nuxt3: https://stackblitz.com/edit/nuxt-starter-gf357r?file=app.vue
nuxt2: https://stackblitz.com/edit/nuxt-starter-drw2fy?file=layouts%2Fdefault.vue
angular: https://stackblitz.com/edit/stackblitz-starters-etu1at?file=src%2Fmain.ts
vanilla: https://stackblitz.com/edit/vitejs-vite-wfnb1k?file=src%2Fmain.ts
svelte kit next: https://stackblitz.com/edit/svelte-kit-9cqx23?file=src%2Froutes%2F%24layout.svelte
svelte kit 2.x: https://stackblitz.com/edit/sveltejs-kit-template-default-qhiudx?file=src%2Froutes%2F%2Blayout.svelte
astro: https://stackblitz.com/edit/withastro-astro-eeghpe?file=src%2Flayouts%2FLayout.astro&title=Astro%20Starter%20Kit:%20Basics
iife: https://stackblitz.com/edit/js-y7xfw5?file=index.html
multiple bind watermark: https://stackblitz.com/edit/vue2-vite-starter-a274yz?file=src%2FApp.vue
on-demand: https://stackblitz.com/edit/webpack-webpack-js-org-wq26h43z?file=src%2Findex.js

**Comment by guhan121** [comment]
how to add watermark for xterm.js
